core/templates/pages/about-page/about-page.component.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the about page.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
+import { WindowRef } from
+  'services/contextual/window-ref.service.ts';
+
+require(
+  'components/common-layout-directives/common-elements/' +
+  'background-banner.directive.ts');
+import { TranslateService } from '@ngx-translate/core';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+
+@Component({
+  selector: 'about-page',
+  template: require('./about-page.component.html'),
+  styleUrls: []
+})
+export class AboutPageComponent implements OnInit {
+  listOfNames: string;
+  aboutPageMascotImgUrl: string;
+  activeTabName: string;
+  listOfNamesToThank = [
+    'Alex Kauffmann', 'Allison Barros',
+    'Amy Latten', 'Brett Barros',
+    'Crystal Kwok', 'Daniel Hernandez',
+    'Divya Siddarth', 'Ilwon Yoon',
+    'Jennifer Chen', 'John Cox',
+    'John Orr', 'Katie Berlent',
+    'Michael Wawszczak', 'Mike Gainer',
+    'Neil Fraser', 'Noah Falstein',
+    'Nupur Jain', 'Peter Norvig',
+    'Philip Guo', 'Piotr Mitros',
+    'Rachel Chen', 'Rahim Nathwani',
+    'Robyn Choo', 'Tricia Ngoon',
+    'Vikrant Nanda', 'Vinamrata Singal',
+    'Yarin Feigenbaum'];
+  // Define constant for each tab on the page.
+  TAB_ID_ABOUT: string = 'about';
+  TAB_ID_FOUNDATION: string = 'foundation';
+  TAB_ID_CREDITS: string = 'credits';
+  ALLOWED_TABS: Array<string>;
+  constructor(
+    private i18nLanguageCodeService: I18nLanguageCodeService,
+    private urlInterpolationService: UrlInterpolationService,
+    private translate: TranslateService,
+    private windowRef: WindowRef) {
+    translate.setDefaultLang('en');
+    this.activeTabName = this.TAB_ID_ABOUT;
```

Comment:
Just one question -- which is a better place for this line and the line below this, inside the constructor or ngOnit? 

(Both constructor and ngOnit run once for a component with constructor running before ngOnit)

Reply:
I just followed their guide: https://github.com/ngx-translate/core#3-init-the-translateservice-for-your-application

Reply:
Umm, I'm not talking about translate.setDefaultLang. I'm talking about this.activeTabName and line below that.

Reply:
Done